### ISSUE TITLE
digitization code using wrong queue

### ISSUE THREAD

**Issue by raphaelcervantes** [axiondarkmatterexperiment/psyllid#3]
I run ` dragonfly cmd -b tpnotc.admx aphid.start-run duration=0 filename="test.egg"`
to run a digitizer. In the log of the digitizer output, I get 

```
^[[1;36m2020-01-18 21:05:25 [DEBUG] (tid 140568322123520) /library/core.cc(356): Using broker authentication: admx:admx^[[0m
^[[1;36m2020-01-18 21:05:25 [DEBUG] (tid 140568239798016) brary/relayer.cc(97): Sending request to <status_message.notice.psyllid>^[[0m
^[[1;36m2020-01-18 21:05:25 [DEBUG] (tid 140568239798016) current_queue.hh(76): Attempting to push to queue^[[0m
^[[1;36m2020-01-18 21:05:25 [DEBUG] (tid 140568239798016) current_queue.hh(78): Pushing to concurrent queue; size: 0^[[0m
```
Notice, that it says `status_message.notice.psyllid` instead of ` status_message.notice.aphid`. 

Either there's a piece of code that has psyllid hard-coded, or that code using the wrong variable, or there’s something wrong with the configuration for aphid. 

However, I ran
`dragonfly cmd -b tpnotc.admx aphid.ping`
to test the queue binding and got the expected message back.

Please fix the code so that it points to the right queue.

More debugging details here: https://maxwell.npl.washington.edu/elog/admx/ADMX-DAQ/210

**Comment by nsoblath** [comment]
The RK to which the alerts are sent is hardcoded, which it clearly shouldn't be.  Thanks for pointing this out.

**Comment by nsoblath** [comment]
@raphaelcervantes I just pushed branch hotfix/1.12.1 that should fix this problem.  are you able to test it out?

**Comment by raphaelcervantes** [comment]
@nsoblath I tested it out and the bug is now fixed. Thank you.

**Comment by nsoblath** [comment]
Fixed in v1.12.1
